### DISCUSSION TITLE
Why MPL license?

### DISCUSSION THREAD

**Discussion by vjeux** [vercel/turborepo#2337] [Ideas]
I’m curious to understand the rationale compared to the more traditional licenses like BSD or MIT.

**Comment by trivikr** [comment]
Looks like this question was asked in Feb'22 in https://github.com/vercel/turbo/discussions/652, and there hasn't been any responses.

**Comment by flexbox** [comment]
Probably because rust itself is in MPL?

<img width="216" alt="image" src="https://user-images.githubusercontent.com/360936/197974674-1a7a3908-51e3-456f-98c8-a8b9f530b0ec.png">


https://github.com/mozilla/rust-parsepatch/blob/master/LICENSE-MPL-2.0

**Comment by nathanhammond** [comment]
MPL, as a copyleft license, helps protect _all_ contributors to the project.
